packages/components/src/query-controls/README.md

```diff
@@ -115,128 +115,116 @@ The format of the categories list also needs to be updated to match what `FormTo
 
--   Type: `Number`
 -   Required: No
 -   Platform: Web
 
-#### categoriesList
+#### `categorySuggestions`: `{ [ categoryName: Entity[ 'name' ] ]: Entity }`
 
-An array of category IDs; renders a `CategorySelect` sub-component when passed in conjunction with `onCategoryChange`.
+An object of categories, renders a `FormTokenField` component when passed in conjunction with `onCategoryChange`.
 
--   Type: `Array`
 -   Required: No
 -   Platform: Web
 
-#### categorySuggestions
+#### `maxItems`: `number`
 
-An array of category names; renders a `FormTokenField` component when passed in conjunction with `onCategoryChange`.
+The maximum of items.
 
--   Type: `Array`
--   Required: No
--   Platform: Web
-
-#### maxItems
-
--   Type: `Number`
 -   Required: No
 -   Default: 100
 -   Platform: Web
 
-#### minItems
+#### `minItems`: `number`
+
+The minimum of items.
 
--   Type: `Number`
 -   Required: No
 -   Default: 1
 -   Platform: Web
 
-#### numberOfItems
+#### `numberOfItems`: `number`
 
 The selected number of items to retrieve via the query.
 
--   Type: `Number`
 -   Required: No
 -   Platform: Web
 
-#### onAuthorChange
+#### `onAuthorChange`: `TreeSelectProps[ 'onChange' ]`
 
 A function that receives the new author value. If this is not specified, the author controls are not included.
 
--   Type: `Function`
 -   Required: No
 -   Platform: Web
 
-#### onCategoryChange
+#### `onCategoryChange`: `TreeSelectProps[ 'onChange' ] & FormTokenFieldProps[ 'onChange' ]`
 
 A function that receives the new category value. If this is not specified, the category controls are not included.
 
--   Type: `Function`
 -   Required: No
 -   Platform: Web
 
-#### onNumberOfItemsChange
+#### `onNumberOfItemsChange`: `( newNumber?: number ) => void`
 
 A function that receives the new number of items value. If this is not specified, then the number of items range control is not included.
 
--   Type: `Function`
 -   Required: No
 -   Platform: Web
 
-#### onOrderChange
+#### `onOrderChange`: `( newOrder: string ) => void`
 
 A function that receives the new order value. If this or onOrderByChange are not specified, then the order controls are not included.
 
--   Type: `Function`
 -   Required: No
 -   Platform: Web
 
-#### onOrderByChange
+#### `onOrderByChange`: `( newOrderBy: string ) => void`
 
 A function that receives the new orderby value. If this or onOrderChange are not specified, then the order controls are not included.
 
--   Type: `Function`
 -   Required: No
 -   Platform: Web
 
-#### order
+#### `order`: `'asc' | 'desc'`
 
-The order in which to retrieve posts. Can be 'asc' or 'desc'.
+The order in which to retrieve posts.
 
--   Type: `String`
 -   Required: No
 -   Platform: Web
 
-#### orderBy
+#### `orderBy`: `'data' | 'title'`
 
-The meta key by which to order posts. Can be 'date' or 'title'.
+The meta key by which to order posts.
+
+-   Required: No
+-   Platform: Web
+
+#### `selectedAuthorId`: `Entity[ 'id' ]`
```

Comment:
This is actually passed as a `number`, not a `string`:

<img width="966" alt="Screen Shot 2022-12-21 at 4 29 52 PM" src="https://user-images.githubusercontent.com/4063887/209031698-0308f076-95b8-4f47-bf43-cf35900aa9e8.png">

Reply:
I'm going to do a little investigation about `Entity` and make sure that `id` is supposed to be a `number`, just to make sure